rules/source-control-tools/rule.md

```diff
@@ -0,0 +1,143 @@
+---
+type: rule
+title: Do you know some Source Control tools ?
```

Comment:
Hi @PothieuG 

- [ ] 1. Please remove the extra space before " ?"
- [ ] 2. I think saying "Do you know some..." sound vague... Can you change to "Do you use..."